src/test/ui/did_you_mean/issue-37139.nll.stderr

```diff
@@ -1,6 +1,8 @@
 error[E0596]: cannot borrow immutable item `x` as mutable
   --> $DIR/issue-37139.rs:22:18
    |
+LL |         TestEnum::Item(ref mut x) => {
+   |                        --------- consider changing this to `mut x`
```

Comment:
yeah we definitely don't want to do this...

Reply:
Yes, need to look at this more carefully.